src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -167,6 +167,14 @@ class GcsPlacementGroupManager : public rpc::PlacementGroupInfoHandler {
       rpc::WaitPlacementGroupUntilReadyReply *reply,
       rpc::SendReplyCallback send_reply_callback) override;
 
+  /// Register a callback of placement group creation.
```

Comment:
The function name and description are inconsistent.

Reply:
yeah, that's a good point, I have updated it.